src/wallpaper.c

```diff
@@ -189,8 +189,9 @@ handle_set_wallpaper_in_thread_func (GTask *task,
            * apps for which an app ID can't be determined.
            */
           g_assert (xdp_app_info_is_host (request->app_info));
-          title = g_strdup (_("Allow Applications to Set Backgrounds?"));
-          subtitle = g_strdup (_("An application is requesting to be able to change the background image."));
+          title = g_strdup (_("Allow to Set Backgrounds?"));
```

Comment:
```suggestion
          title = g_strdup (_("Allow an application to Set the Backgrounds?"));
```

Reply:
We are not granting the permission to just one application, as per comment above:
```
/* Note: this will set the wallpaper permission for all unsandboxed
 * apps for which an app ID can't be determined.
 */
````